hugo/content/exporter/_index.md

```diff
@@ -264,23 +264,23 @@ sudo systemctl status crunchy-postgres-exporter@postgres_exporter_pg##_per_db
 
 ### Monitoring multiple databases and/or running multiple postgres exporters (RHEL / CentOS)
 
-Certain metrics are not cluster-wide, so multiple exporters must be run to collect all relevant metrics. As of v0.5.x of postgres_exporter, a single service can connect to multiple databases. As long as you're using the same custom query file for all of those databases, only one additional exporter service is required and this comes with pgMonitor 4.0 and above by default. The {{< shell >}}queries_per_db.yml{{< /shell >}} file contains these queries and the secondary exporter can use this file to collect those metrics and avoid duplicating cluster-wide metrics. Note that some other metrics are per database as well (Ex. bloat). You can then define multiple targets for that one job in Prometheus so that all the metrics are collected together for a single database instance. Note that the "setup.sql" file does not need to be run on these additional databases if using the queries that pgMonitor comes with.
+Certain metrics are not cluster-wide, so multiple exporters must be run to collect all relevant metrics. As long as you're using the same custom query file for all monitored databases, only one additional exporter service is required and pgMonitor does this for collecting its per-database metrics ({{< shell >}}queries_per_db.yml{{< /shell >}}). In Prometheus you can then define the global and per-db exporter targets for a single job so that all the metrics are collected together for a single database instance. Note that the "setup.sql" file does not need to be run on these additional databases if using the queries that pgMonitor comes with.
```

Comment:
```suggestion
Certain collected metrics are cluster-wide while others are per-database. As such, a separate exporter needs to be run to collect both kinds of metrics. Furthermore, if you are not collecting _the same_ metrics in all your databases, you will need one exporter per group of collected metrics. Using the shipped {{< shell >}}queries_per_db.yml{{< /shell >}} for all your per-database metrics ensures that you only need one exporter for cluster-wide metrics and one exporter for per-db metrics _regardless of the number of monitored databases_. By default, pgMonitor will be set up this way. In Prometheus, you can then define the global and per-db exporter targets for a single job. This will place all the metrics that are collected for a single databse instance together.

{{< note >}}The "setup.sql" file does not need to be run on these additional databases if using the queries that pgMonitor comes with.{{< /note >}}
```
?

Reply:
First sentence as rewritten isn't really true. You can collect cluster-wide and per-db metrics in a single exporter. The issue is that if you then try and monitor multiple databases with that, you get duplicated cluster-wide metrics.

Reply:
@hunleyd I've rewritten this part again. Hopefully less wordy but more clear